### PR TITLE
Theme 15 v6 components html5

### PR DESCRIPTION
**Pull request by cjwest**
# READY FOR REVIEW

# Summary
- Replaces https://github.com/SU-SWS/decanter/pull/474 and https://github.com/SU-SWS/decanter/pull/481
- Please see conversation at https://github.com/SU-SWS/decanter/pull/474
- Proposing HTML5 syntax and role=alert to the alert component

# Needed By (Date)

# Urgency
- Not Urgent

# Steps to Test

1. Check out this branch
2. Review and comment on changes
3. Verify the alert component works as expected.

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- JIRA ticket- https://stanfordits.atlassian.net/browse/THEME-15
- Other PRs: This takes the place of https://github.com/SU-SWS/decanter/pull/474
- I rebuilt this PR to merge into V6 instead of Master. 
- Anyone who should be notified? 
@sherakama 
@yvonnetangsu 
@rebeccahongsf 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by cjwest** [comment]
@JBCSU,
I'd love your $0.02 as I'm learning about how to use HTML5.
What do you think, should we use <article> or <div> to wrap the alert component?

Please see discussions:
https://github.com/SU-SWS/decanter/pull/481
https://github.com/SU-SWS/decanter/pull/474

File: core/src/templates/components/alert/alert.twig [hunk 1]
```diff
@@ -19,13 +19,13 @@
  * - alert_body: Text details of the alert message.
  */
 #}
-<div {{ attributes }} class="su-alert {{ modifier_class }}" >
-  <div class="su-alert__body">
+<article {{ attributes }} class="su-alert {{ modifier_class }}">
```
Comment (sherakama) [inline core/src/templates/components/alert/alert.twig]: After the discussion in #474 I think `<div>` is the appropriate element after all.

**Comment by yvonnetangsu** [comment]
> @JBCSU,
> I'd love your $0.02 as I'm learning about how to use HTML5.
> What do you think, should we use `<article>` or `<div>` to wrap the alert component?
> 
> Please see discussions:
> #481
> #474

Hi @cjwest , I believe JB replied here: https://github.com/SU-SWS/decanter/pull/481
She prefers using div.

**Comment by JBCSU** [comment]
> @JBCSU,
> I'd love your $0.02 as I'm learning about how to use HTML5.
> What do you think, should we use `<article>` or `<div>` to wrap the alert component?

I prefer `<div>`.

**Comment by cjwest** [comment]
@sherakama, @JBCSU, and @yvonnetangsu 
Okay, the `<div>` it is!
I'll close this PR and we'll continue with what already exists.
Thank you everyone for the great discussion! It's really helpful to have a chance to get ideas and feedback on how the HTML5 elements are to be used.

**Pull request closed without merging**